packages/synapse-react-client/src/components/AccessTokenPage/AccessTokenCard.tsx

```diff
@@ -104,7 +104,8 @@ export const AccessTokenCard: React.FunctionComponent<AccessTokenCardProps> = ({
       {/* Delete button */}
       <div className="PersonalAccessTokenCard__DeleteButton">
         <Button
-          variant="default"
+          variant="outlined"
+          color="error"
```

Comment:
Updated to error color --  
<img width="1181" alt="updated_AccessTokenCard" src="https://github.com/Sage-Bionetworks/synapse-web-monorepo/assets/26949006/a4a28943-4954-4982-b1a7-47dc3d51620f">